hypervisor/src/cpu.rs

```diff
@@ -449,6 +449,11 @@ pub trait Vcpu: Send + Sync {
     /// Triggers the running of the current virtual CPU returning an exit reason.
     ///
     fn run(&self) -> std::result::Result<VmExit, HypervisorCpuError>;
+    #[cfg(all(feature = "kvm", target_arch = "x86_64"))]
+    ///
+    /// Translate guest virtual address to guest physical address
+    ///
+    fn translate_gva(&self, gva: u64) -> Result<u64>;
```

Comment:
That sounds better, but it needs to remove the `mshv` specific return type from `translate_gva()`. I made the required change as commit [a48ae41](https://github.com/cloud-hypervisor/cloud-hypervisor/pull/3575/commits/a48ae418e22597982702c4ba7f91b4b71f0b9210) so that the subsequent commit can implement the function for `kvm` ([a717192](https://github.com/cloud-hypervisor/cloud-hypervisor/pull/3575/commits/a717192c76df84bc414ef67de0e3aa3b2d2d98f4)).